server/utils/queryHelpers/scopeGet.js

```diff
@@ -68,6 +68,17 @@ const getActiveIds = ({ activePub, activeCollection, activeCommunity }) => {
 	};
 };
 
+const getActivePubQuery = ({ id, slug, communityId }) => {
+	if (!communityId && slug) {
+		throw new Error('Pub slug must be passed to scopeGet with a communityId');
+	}
```

Comment:
I'm not sure this is even needed (nor the older code on the `activePub` query below). `slug` is constrained to be unique across the Pubs table, so there's no need to specify the `communityId` regardless of whether you have `id` or `slug`.

Reply:
Cool — I will remove this check.